deploy/crds/crd-certificaterequests.yaml

```diff
@@ -34,7 +40,7 @@ spec:
       service:
         # If you have deployed cert-manager into a namespace other than
         # 'cert-manager', be sure to update this value.
-        namespace: cert-manager
+        namespace: '{{ .Release.Namespace }}'
         name: cert-manager-webhook
```

Comment:
TODO: I've not updated the service name used here to be templated by Helm, but we should do that :)